www/src/js/models/snap.js

```diff
@@ -166,10 +166,6 @@ module.exports = Backbone.Model.extend({
       response.icon = this.defaults.icon;
     }
 
-    if (response.hasOwnProperty('developer') && !response.developer.length) {
-      response.developer = this.defaults.developer;
-    }
-
```

Comment:
The result is just an empty element when Handlebars renders the template, so:

```html
<div class="b-snap__developer"></div>
```

instead of:

```html
<div class="b-snap__developer">-</div>
```